portfolio/src/main/webapp/style.css

```diff
@@ -5,9 +5,44 @@
   padding: 50px;
   margin: 20px;
   position: relative;
-  left:20%;
+  left: 20%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* Add a background to the top of the webpage */
+#topHeader {
+    width: 1000px;
+    height: 160px;
+    position: relative;
+    left: 15%;
+}
+
+/* Customize the background image */
+#topHeader img {
+    width: 1000px;
+    height: 160px;
+}
+
+/* Create and customize a navigation bar at the top of the webpage */
+.navigation {
+    width: 1000px;
+    overflow: hidden;
+    background-color: skyblue;
+    position: relative;
+    left: 15%;
+}
+
+.navigation a {
+    float: left;
+    display: block;
+    background-color: dodgerblue;
+    color: black;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
```

Comment:
Curious, is 'none' the default value? Can we remove this line?

Reply:
I may think the text-decoration is not none by default. Because, it's a link. The default text-decoration for a link is underline. I don't want that property, so I set it to none.